docs/tctl/workflow/describeid.md

```diff
@@ -10,9 +10,9 @@ tags:
 
 The `tctl workflow describeid` command shows information about a [Workflow Execution](/concepts/what-is-a-workflow-execution) for the specified [Workflow Id](/concepts/what-is-a-workflow-id) and optional [Run Id](/concepts/what-is-a-run-id).
 
-`tctl workflow describeid <workflow_id> [<run_id>] [<modifiers>]`
+`tctl workflow describeid <workflow_id> <run_id> <modifiers>`
```

Comment:
The square brackets designated portions of the syntax line that are optional. Having said that, I'm not sure how clear the designation is, so I'm fine with removing them.